src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1041,4 +1041,7 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Warning As Error: {0}</value>
     <comment>{0} is a warning message</comment>
   </data>
+  <data name="SourcesCommandValidProtocolVersion" xml:space="preserve">
+    <value>The protocol version specified is invalid. Provide a valid protocol version.</value>
```

Comment:
Should we just list what are valid versions here?

Reply:
I've updated the error message to list versions which are valid.